modules/broker-auth/src/main/java/io/ballerina/messaging/broker/auth/AuthManager.java

```diff
@@ -130,6 +145,15 @@ public boolean isAuthenticationEnabled() {
         return brokerAuthConfiguration.getAuthentication().isEnabled();
     }
 
+    /**
+     * Provides broker authorization enabled
+     *
+     * @return broker authorization enabled or not
+     */
+    public boolean isAuthorizationEnabled() {
+        return brokerAuthConfiguration.getAuthorization().isEnabled();
```

Comment:
Shall we keep this as a final field since it should not change after restart?

Reply:
Fixed in d1ed2c6